src/bin/tikv-fail.rs

```diff
@@ -49,6 +49,7 @@ fn main() {
         .about("A tool for injecting failures to TiKV and recovery")
         .arg(
             Arg::with_name("addr")
+                .required(true)
```

Comment:
please use another PR for this change :-)

Reply:
https://github.com/pingcap/tikv/pull/2602
i have make another PR here